EmeraldComponents/Components/Picker/GenericPicker/BasePickerViewController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  BasePickerViewController.swift
+//  EmeraldComponents
+//
+//  Created by Daniele  Freitas on 23/12/19.
+//  Copyright © 2019 StoneCo. All rights reserved.
+//
+
+import UIKit
+/**
+ Class that is the base for the generic picker view components
+ Controls the input and output for picker component with generic type (Any)
+ */
+class BasePickerViewController: UIViewController {
```

Comment:
Tem que lembrar de deixar public essas classes se for usar fora do framework.
```suggestion
public class BasePickerViewController: UIViewController {
```